MergerLogic/DataTypes/Data.cs

```diff
@@ -34,6 +34,8 @@ public enum GridOrigin
 
     public abstract class Data<TUtilsType> : IData where TUtilsType : IDataUtils
     {
+        protected readonly int MaxZoomRead = 30;

```

Comment:
if it's used only in the S3 section - define it there

Reply:
why not const?
and why 30???
~72micron is insane resolution for map and un storable large
23 is already less then 1cm and more them we will EVER need for map